### DISCUSSION TITLE
Looking for pointers or something ........

### DISCUSSION THREAD

**Discussion by kernel-task** [Stellarium/stellarium#2010] [Q&A] [answered]
Attached is a screenshot from an interactive system I have that generates charts used for observation planning.
(Kind of a year at a glance!) It displays numerous bits and pieces of data the details of which are probably uninteresting to most.
Among the plotted data are transit lines for various celestial objects (the dotted lines in the attached example). The raw transit data is generated using the calculation tools within Stellarium. The output text files are then taken into my system.
What I can't currently do is generate transit data for a "random" point on the celestial sphere (e.g. say First Point of Aries). (Stellarium does state as much which is fine, I can accept that) but would any user, more experienced than I, be able to tell me if there might be a way to generate transit data from a set of coordinates rather than a selected body.
A general outline would do, I do not expect to have my problem solved for me just directions I might look in.

Thanks in advance for any thoughts/assistance.
Regards.
Frank C.

<img width="1112" alt="Screenshot 2021-11-04 at 13 05 53" src="https://user-images.githubusercontent.com/16398788/140321061-d3e056b1-a147-4bad-af38-7bbf9aba3173.png">

**Comment by alex-w** [comment, marked as answer]
Please use custom markers (F1 for details)

**Reply by alex-w** [reply]
In theory we may add similar graph into AstroCalc tool for major planets, moon and one selected object.

**Reply by kernel-task** [reply]
Outstanding!  Perfect solution.
Regards.
FC

**Reply by kernel-task** [reply]
If you do add some such graph I would (personally) advise centring the time axis it on midnight rater than midday (I have been unable to do so - a gnuplot! issue)
The observation window (area between astronomical twilights) then becomes an oval shape in the centre of the graph. This outlines a sort of 'optimal' viewing window.

Don't know if I am making sense but.....

Again thanks for **actually** solving my problem.
Regards
FC
